src/Features/CSharp/Portable/ImplementInterface/AbstractChangeImplementationCodeRefactoringProvider.cs

```diff
@@ -231,10 +231,17 @@ private MemberImplementationMap GetImplementedMembers(INamedTypeSymbol containin
             var documentToImplDeclarations = new OrderedMultiDictionary<Document, (SyntaxNode, ISymbol impl, SetWithInsertionOrder<ISymbol> interfaceMembers)>();
             foreach (var (implMember, interfaceMembers) in implMemberToInterfaceMembers)
             {
-                foreach (var syntaxRef in implMember.DeclaringSyntaxReferences)
+                foreach (var location in implMember.Locations)
                 {
-                    var doc = solution.GetRequiredDocument(syntaxRef.SyntaxTree);
-                    var decl = await syntaxRef.GetSyntaxAsync(cancellationToken).ConfigureAwait(false);
+                    if (location.SourceTree is null)
+                        continue;
+
+                    var doc = solution.GetRequiredDocument(location.SourceTree);
+                    var declToken = location.FindToken(cancellationToken);
+                    var decl = GetContainer(declToken).declaration;
```

Comment:
we weren't verifying before that for the *other* members we wanted to update we were able to get a viable declaration to update.